src/NServiceBus.Core/Transports/ErrorContext.cs

```diff
@@ -22,7 +22,12 @@ public class ErrorContext
         /// <summary>
         /// Number of immediate processing attempts.
```

Comment:
I think we can be more precise here and say: "Number of failed immediate processing attempts. This number is reset after each delayed retry."